plotter/plotter.go

```diff
@@ -127,8 +127,11 @@ func XYRange(xys XYer) (xmin, xmax, ymin, ymax float64) {
 	return
 }
 
+// XY is an x and y value.
+type XY struct{ X, Y float64 }
+
 // XYs implements the XYer interface.
-type XYs []struct{ X, Y float64 }
+type XYs []XY
```

Comment:
to prevent breaking the flow of someone new to `gonum/plot` and reading the code, I would perhaps put the definition of `XY` _after_ the one of `XYs`.

Reply:
Yes, that is the convention. It's not a huge issue here though since the types are so close to each other in the text.